### PR TITLE
nav background updates

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

File: network-api/networkapi/templates/pages/primary_nav.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{% if page.zen_nav == True %}<div id="primary-nav-container" data-nav-mode="zen" class="zen-mode">
```
Comment (ScottDowne) [inline network-api/networkapi/templates/pages/primary_nav.html]: I think I lost some changes in here during the rebase, I need to re copy it to ensure it's not regressing anything.

Comment (gvn) [reply network-api/networkapi/templates/pages/primary_nav.html]: Have you checked this?

Comment (ScottDowne) [reply network-api/networkapi/templates/pages/primary_nav.html]: yup, the conflict was due to internet health links updates. Both links should go to "/internet-health"

**Comment by gvn** [comment]
@ScottDowne You've got merge conflicts here ^

**Pull request merged**